ckcp/openshift_dev_setup.sh

```diff
@@ -76,8 +71,33 @@ parse_args() {
 }
 
 init() {
-  APP_LIST="openshift-gitops compute ckcp register-compute"
-  cluster_type="openshift"
+
+  APP_LIST=(
+            "openshift-gitops" 
+            "ckcp"
+           )
+  #get the APPS to be installed
+  read -ra APPS <<< "$(yq eval '.APPS | join(" ")' "$DEFAULTS")"
+  for app in  "${APPS[@]}"
+  do
+    APP_LIST+=("$app")
+  done
+
+  #get cluster type
+  cluster_type=$(yq '.CLUSTER_TYPE' "$DEFAULTS")
+  cluster_type=${cluster_type:-"openshift"}
+
+  GIT_URL=$(yq '.GIT_URL' "$DEFAULTS")
+  GIT_REF=$(yq '.GIT_REF' "$DEFAULTS")
+  # set default values in case they're missing
+  GIT_URL=${GIT_URL:-"https://github.com/openshift-pipelines/pipelines-service.git"}
+  GIT_REF=${GIT_REF:="main"}
+
+  # get values of CR_TO_SYNC
+  read -ra CR_TO_SYNC <<< "$(yq eval '.CR_TO_SYNC | join(" ")' "$SCRIPT_DIR/defaults.yaml")"
```

Comment:
`"$SCRIPT_DIR/defaults.yaml"` -> `"$DEFAULTS"` (although you might want to rename the var if you rename the file.

Reply:
done.